.circleci/config.yml

```diff
@@ -316,6 +364,50 @@ jobs:
       - store_test_results:
           path: reports/
     resource_class: large
+  test_similarity_api:
+    executor: docker-python-executor
+    steps:
+      - attach_workspace:
+          at: .
+      - setup_remote_docker:
+          version: 20.10.14
+      - run:
+          name: Syft SBOM
+          environment:
+            SYFT_VERSION: v0.82.0
+            IMAGE_NAME: ghcr.io/kcirtapfromspace/cloudfoundry_circleci:latest
```

Comment:
Does this image require Pat's repo to exist? If so, we may want to fork it and give it to adhoc or HHS

Reply:
(not that he'll leave us high and dry)

Reply:
Yeah, it does. Forking it is probably a good idea for continuity's sake.

Reply:
Haven't looked  at most of the changes, but this image reference is more aligned with deploying a docker container to cloud.gov. I believe I recreated this functionality with the CI/CD using the docker-compose, similar to how the js app performs its vulnerability scan.

Reply:
check out if this is still relevant https://github.com/HHS/Head-Start-TTADP/pull/1561/files#diff-2970c60e47e213e52aa3fb4cfdf0c093e9cd3877576dc990aa046904938f03d0R35-R51

Reply:
Here is the SBOM & coverage
https://github.com/HHS/Head-Start-TTADP/pull/1561/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R355-R407